oeps/oep-0002-bp-repo-metadata.rst

```diff
@@ -37,19 +37,28 @@ Bear in mind that particular classes of Open edX repositories may have different
 Specification
 =============
 
+Keys
+----
+
 Each repo will include a file ``openedx.yaml``, with the following keys:
 
-``owner``: string (required unless ``archived`` is ``True``)
-    The GitHub user responsible for the repository.
+``owner``: dictionary (required)
```

Comment:
Nit: do we care if the teams/repos are prefixed with `edx/` or not? If the answer is "no we don't care" and we're fine with tools handling both, that's fine with me.

Reply:
Good point.  I've added prefixing with the Github organization.